### DISCUSSION TITLE
Para quem quiser Adicionar o Botão Globo Play

### DISCUSSION THREAD

**Discussion by ronaldorj** [PRProd/HA-Firemote#272] [Ideas]
Basta Adicionar o código a baixo no js HA-Firemoto.js:

"globo-tv": {
      "button": '<svg id="Camada_1" data-name="Camada 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 753.9 163.7"> <defs> <linearGradient id="Gradiente_sem_nome_19" x1="11.036" y1="100.444" x2="731.818" y2="100.444" gradientUnits="userSpaceOnUse"> <stop offset="0.123" stop-color="#fe012f"/> <stop offset="0.955" stop-color="#fe8e04"/> </linearGradient> <linearGradient id="Gradiente_sem_nome_19-2" x1="11.611" y1="101.241" x2="733.251" y2="101.241" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-3" x1="14.01" y1="64.448" x2="728.927" y2="64.448" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-4" x1="13.579" y1="99.265" x2="731.751" y2="99.265" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-5" x1="13.717" y1="82.231" x2="732.005" y2="82.231" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-6" x1="11.074" y1="82.149" x2="732.773" y2="82.149" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-7" x1="11.053" y1="82.049" x2="732.799" y2="82.049" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-8" x1="38.364" y1="64.206" x2="723.469" y2="64.206" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-9" x1="14.727" y1="64.02" x2="712.891" y2="64.02" xlink:href="#Gradiente_sem_nome_19"/> </defs> <path d="M41.3,164.3c-17.2,0-30-6-36.7-11.3-3.7-3-4.4-5.3-2.4-9,1.4-2.6,2.7-5.3,4-7.9,2.3-4.6,4.5-5.4,9.1-2.7,8.7,5.1,17.8,8.2,28.2,7.8,16.1-.3,22.2-10.4,21.8-24.3-3.5,4.1-10.4,8.7-22.7,8.8-8.9,0-18.8-3.3-25.3-8.3C3.3,106.9-1.9,92.1.7,75.4c2.8-18.5,13.4-31.6,32-37.2,12.6-3.8,24.3-.9,34.6,8.9-.1-4,.3-6.5,2-7.8,1.7-1.2,5.6-1.3,8-1.2,12.9.1,12.6-.5,12.5,8.4-.1,18.5.2,45-.2,63.5C89.8,134,82.2,164.3,41.3,164.3ZM66.3,82c.1-13.4-8.7-22.6-21.2-22.9-12.4-.3-22.1,8.6-22,22.7.1,14.7,9.3,23,21.9,23.1C56.8,105.1,66.4,96.1,66.3,82Z" style="fill:url(#Gradiente_sem_nome_19)"/> <path d="M666.5,158c-3.1-1.6-2.2-4.8-.7-7.6.5-1,1.1-2,1.7-3,4.1-8,4.1-8.1,12.6-6.1,5.4,1.2,9.7.1,12.6-4.8a91.025,91.025,0,0,0,7-13.5s-23-52.9-33.5-76.8c-3-6.8-2.7-7.2,4.6-7.2,4.7,0,9.3.2,14-.1,3.4-.2,5,1.3,6.2,4.3,6.2,15.8,19.6,50.5,19.6,50.5,2.6-7.5,5.5-15.2,7.9-21.9,3.3-9.4,6.8-18.7,9.8-28.2,1.2-3.6,3.1-5,6.9-4.8,4.3.3,8.6.1,13,.1,5.9,0,6.9,1.6,4.7,7.1-5.3,13.3-10.6,26.7-16,40-6.6,16.5-14.8,36.6-21.6,53-4.1,9.9-14,24.7-30.2,24.7C673.7,163.7,670.7,160.2,666.5,158Z" style="fill:url(#Gradiente_sem_nome_19-2)"/> <path d="M258.7,45.9c5.2-5.6,15.1-9.2,23-9.2,11.7,0,23.4,4.6,33.1,16.5,12.4,15,12.5,44.7-1.2,59.6-7.5,8.7-16.2,14.6-30.7,14.6A37.119,37.119,0,0,1,257,117.3c-.6-.5-.2,4.9-1.2,6.6-1.5,2.8-18.4,2.6-20.1-.2a8.6,8.6,0,0,1-.9-4.2c0-36.9,0-74.9-.1-111.8,0-4.5,2-6.3,6.3-6.2,3.7.1,7.3.3,11,.1,4.8-.2,6.4,2.1,6.4,6.6C258.4,19.1,258.6,45.5,258.7,45.9Zm20.6,13.3c-12-.1-21.2,8.9-21.5,22.8-.3,13.2,9.4,23.3,21.6,23.3,12.2-.1,21.3-9.9,21.4-23C300.9,68.8,291.9,59.2,279.3,59.2Z" style="fill:url(#Gradiente_sem_nome_19-3)"/> <path d="M457.2,118.1c0,12.4.1,23.8,0,35.2-.1,8.4-.2,8.4-8.7,8.4-3,0-6-.2-9,0-3.9.2-5.2-1.8-5.2-5.4V44.1c0-3.7,1.4-5.6,5.3-5.3,2.5.2,7,.1,9.5,0,6.9-.1,6.5,2,6.5,8.6,24.1-22.9,59.6-5.7,66,23.6,4.8,22-4.5,42.9-23.2,52.2-12.3,6-24.4,5.9-36.1-1.7C460.9,120.5,459.6,119.6,457.2,118.1Zm-.4-36.2c-.1,13.1,9.3,23.5,21.6,23.3,12.9-.2,21.9-9.6,21.7-23.1-.2-13-7.1-22.4-20.7-22.9C466.2,58.7,456.9,68.4,456.8,81.9Z" style="fill:url(#Gradiente_sem_nome_19-4)"/> <path d="M634.1,46.9c1.1-8.1-.1-8.2,8.9-8.1,2.7,0,5.3.2,8,0,4.5-.5,5.9,2,5.9,5.9v75c0,4.2-1.8,6.5-6.4,6.1-2.3-.2-4.7,0-7,0-8,0-9.6.1-9.6-8.8-4.9,5.9-13.1,10-23.1,10.5-12.4-.3-17.9-2.4-26.4-9.3-23.9-19.5-21.9-58,4-74.8,13.1-8.5,29.2-9,42,1C631.4,45.1,632.5,45.8,634.1,46.9Zm-43,35c0,13.6,9,23.7,21.3,23.4C625,105,633.9,95.9,634,82c.1-15.1-11.5-23-21.8-23.1C599.6,58.9,589.6,71.2,591.1,81.9Z" style="fill:url(#Gradiente_sem_nome_19-5)"/> <path d="M224.6,81.9c-.2,26.1-19.2,45.5-44.9,45.6-25.9.1-45.3-20.2-44.9-46.4.3-23.2,19.2-44.6,45.1-44.3C206.7,37.1,224.5,57.8,224.6,81.9Zm-23.3,1c.3-13-8.7-23.4-20.5-23.7-12.9-.3-22.7,9-23.1,21.9-.4,13.1,9.3,24,21.5,24.2S200.9,95.7,201.3,82.9Z" style="fill:url(#Gradiente_sem_nome_19-6)"/> <path d="M333.5,81.8c0-27,20.7-45.2,44.9-44.9,27.4.3,45.2,20.7,44.6,46.2s-19.4,44.3-45.3,44.1C352.4,127.1,333,107.9,333.5,81.8Zm66.4.1c-.1-13.6-9.2-22.8-22.5-22.8-11.4-.1-22,10.1-21.2,24.7a22.5,22.5,0,0,0,22.6,21.5C390.9,105.1,400,95,399.9,81.9Z" style="fill:url(#Gradiente_sem_nome_19-7)"/> <path d="M557,64.3c0,18.5-.1,37.1.1,55.6,0,4.3-1.6,6-5.8,5.8a100.9,100.9,0,0,0-11,0c-4.8.3-6.6-2-6.6-6.5V10c0-6.3.9-7.2,7.1-7.3,3.5,0,7,.1,10.5,0,4.3-.2,5.8,1.7,5.8,5.9C556.9,27.2,557,45.8,557,64.3Z" style="fill:url(#Gradiente_sem_nome_19-8)"/> <path d="M101.1,64.2V7.1c0-6.8,2.7-9.1,11.1-5.4,9.7,4.2,12.2,5.2,12.2,13.1,0,30.9,0,65.2-.3,97.1-.1,6.2.3,8.9-7.3,12.3-1.6.7-4.7,2.3-7,3.2-4.6,1.9-8.8,0-8.7-6.7.1-13.4,0-26.8-.1-40.2,0-5.4,0-10.8.1-16.3Z" style="fill:url(#Gradiente_sem_nome_19-9)"/></svg>',
      "friendlyName": "globo TV",
      "className": "globoPlayButton",
      "appName": "com.globo.globotv",
      "androidName": "com.globo.globotv",
      "adbLaunchCommand": "adb shell am start -a android.intent.action.MAIN -n com.globo.globotv/.maintv.MainActivity",
      "deviceFamily": ["amazon-fire"], },

**Comment by ronaldorj** [comment]
svg do botão:
`<svg id="Camada_1" data-name="Camada 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 753.9 163.7"> <defs> <linearGradient id="Gradiente_sem_nome_19" x1="11.036" y1="100.444" x2="731.818" y2="100.444" gradientUnits="userSpaceOnUse"> <stop offset="0.123" stop-color="#fe012f"/> <stop offset="0.955" stop-color="#fe8e04"/> </linearGradient> <linearGradient id="Gradiente_sem_nome_19-2" x1="11.611" y1="101.241" x2="733.251" y2="101.241" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-3" x1="14.01" y1="64.448" x2="728.927" y2="64.448" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-4" x1="13.579" y1="99.265" x2="731.751" y2="99.265" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-5" x1="13.717" y1="82.231" x2="732.005" y2="82.231" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-6" x1="11.074" y1="82.149" x2="732.773" y2="82.149" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-7" x1="11.053" y1="82.049" x2="732.799" y2="82.049" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-8" x1="38.364" y1="64.206" x2="723.469" y2="64.206" xlink:href="#Gradiente_sem_nome_19"/> <linearGradient id="Gradiente_sem_nome_19-9" x1="14.727" y1="64.02" x2="712.891" y2="64.02" xlink:href="#Gradiente_sem_nome_19"/> </defs> <path d="M41.3,164.3c-17.2,0-30-6-36.7-11.3-3.7-3-4.4-5.3-2.4-9,1.4-2.6,2.7-5.3,4-7.9,2.3-4.6,4.5-5.4,9.1-2.7,8.7,5.1,17.8,8.2,28.2,7.8,16.1-.3,22.2-10.4,21.8-24.3-3.5,4.1-10.4,8.7-22.7,8.8-8.9,0-18.8-3.3-25.3-8.3C3.3,106.9-1.9,92.1.7,75.4c2.8-18.5,13.4-31.6,32-37.2,12.6-3.8,24.3-.9,34.6,8.9-.1-4,.3-6.5,2-7.8,1.7-1.2,5.6-1.3,8-1.2,12.9.1,12.6-.5,12.5,8.4-.1,18.5.2,45-.2,63.5C89.8,134,82.2,164.3,41.3,164.3ZM66.3,82c.1-13.4-8.7-22.6-21.2-22.9-12.4-.3-22.1,8.6-22,22.7.1,14.7,9.3,23,21.9,23.1C56.8,105.1,66.4,96.1,66.3,82Z" style="fill:url(#Gradiente_sem_nome_19)"/> <path d="M666.5,158c-3.1-1.6-2.2-4.8-.7-7.6.5-1,1.1-2,1.7-3,4.1-8,4.1-8.1,12.6-6.1,5.4,1.2,9.7.1,12.6-4.8a91.025,91.025,0,0,0,7-13.5s-23-52.9-33.5-76.8c-3-6.8-2.7-7.2,4.6-7.2,4.7,0,9.3.2,14-.1,3.4-.2,5,1.3,6.2,4.3,6.2,15.8,19.6,50.5,19.6,50.5,2.6-7.5,5.5-15.2,7.9-21.9,3.3-9.4,6.8-18.7,9.8-28.2,1.2-3.6,3.1-5,6.9-4.8,4.3.3,8.6.1,13,.1,5.9,0,6.9,1.6,4.7,7.1-5.3,13.3-10.6,26.7-16,40-6.6,16.5-14.8,36.6-21.6,53-4.1,9.9-14,24.7-30.2,24.7C673.7,163.7,670.7,160.2,666.5,158Z" style="fill:url(#Gradiente_sem_nome_19-2)"/> <path d="M258.7,45.9c5.2-5.6,15.1-9.2,23-9.2,11.7,0,23.4,4.6,33.1,16.5,12.4,15,12.5,44.7-1.2,59.6-7.5,8.7-16.2,14.6-30.7,14.6A37.119,37.119,0,0,1,257,117.3c-.6-.5-.2,4.9-1.2,6.6-1.5,2.8-18.4,2.6-20.1-.2a8.6,8.6,0,0,1-.9-4.2c0-36.9,0-74.9-.1-111.8,0-4.5,2-6.3,6.3-6.2,3.7.1,7.3.3,11,.1,4.8-.2,6.4,2.1,6.4,6.6C258.4,19.1,258.6,45.5,258.7,45.9Zm20.6,13.3c-12-.1-21.2,8.9-21.5,22.8-.3,13.2,9.4,23.3,21.6,23.3,12.2-.1,21.3-9.9,21.4-23C300.9,68.8,291.9,59.2,279.3,59.2Z" style="fill:url(#Gradiente_sem_nome_19-3)"/> <path d="M457.2,118.1c0,12.4.1,23.8,0,35.2-.1,8.4-.2,8.4-8.7,8.4-3,0-6-.2-9,0-3.9.2-5.2-1.8-5.2-5.4V44.1c0-3.7,1.4-5.6,5.3-5.3,2.5.2,7,.1,9.5,0,6.9-.1,6.5,2,6.5,8.6,24.1-22.9,59.6-5.7,66,23.6,4.8,22-4.5,42.9-23.2,52.2-12.3,6-24.4,5.9-36.1-1.7C460.9,120.5,459.6,119.6,457.2,118.1Zm-.4-36.2c-.1,13.1,9.3,23.5,21.6,23.3,12.9-.2,21.9-9.6,21.7-23.1-.2-13-7.1-22.4-20.7-22.9C466.2,58.7,456.9,68.4,456.8,81.9Z" style="fill:url(#Gradiente_sem_nome_19-4)"/> <path d="M634.1,46.9c1.1-8.1-.1-8.2,8.9-8.1,2.7,0,5.3.2,8,0,4.5-.5,5.9,2,5.9,5.9v75c0,4.2-1.8,6.5-6.4,6.1-2.3-.2-4.7,0-7,0-8,0-9.6.1-9.6-8.8-4.9,5.9-13.1,10-23.1,10.5-12.4-.3-17.9-2.4-26.4-9.3-23.9-19.5-21.9-58,4-74.8,13.1-8.5,29.2-9,42,1C631.4,45.1,632.5,45.8,634.1,46.9Zm-43,35c0,13.6,9,23.7,21.3,23.4C625,105,633.9,95.9,634,82c.1-15.1-11.5-23-21.8-23.1C599.6,58.9,589.6,71.2,591.1,81.9Z" style="fill:url(#Gradiente_sem_nome_19-5)"/> <path d="M224.6,81.9c-.2,26.1-19.2,45.5-44.9,45.6-25.9.1-45.3-20.2-44.9-46.4.3-23.2,19.2-44.6,45.1-44.3C206.7,37.1,224.5,57.8,224.6,81.9Zm-23.3,1c.3-13-8.7-23.4-20.5-23.7-12.9-.3-22.7,9-23.1,21.9-.4,13.1,9.3,24,21.5,24.2S200.9,95.7,201.3,82.9Z" style="fill:url(#Gradiente_sem_nome_19-6)"/> <path d="M333.5,81.8c0-27,20.7-45.2,44.9-44.9,27.4.3,45.2,20.7,44.6,46.2s-19.4,44.3-45.3,44.1C352.4,127.1,333,107.9,333.5,81.8Zm66.4.1c-.1-13.6-9.2-22.8-22.5-22.8-11.4-.1-22,10.1-21.2,24.7a22.5,22.5,0,0,0,22.6,21.5C390.9,105.1,400,95,399.9,81.9Z" style="fill:url(#Gradiente_sem_nome_19-7)"/> <path d="M557,64.3c0,18.5-.1,37.1.1,55.6,0,4.3-1.6,6-5.8,5.8a100.9,100.9,0,0,0-11,0c-4.8.3-6.6-2-6.6-6.5V10c0-6.3.9-7.2,7.1-7.3,3.5,0,7,.1,10.5,0,4.3-.2,5.8,1.7,5.8,5.9C556.9,27.2,557,45.8,557,64.3Z" style="fill:url(#Gradiente_sem_nome_19-8)"/> <path d="M101.1,64.2V7.1c0-6.8,2.7-9.1,11.1-5.4,9.7,4.2,12.2,5.2,12.2,13.1,0,30.9,0,65.2-.3,97.1-.1,6.2.3,8.9-7.3,12.3-1.6.7-4.7,2.3-7,3.2-4.6,1.9-8.8,0-8.7-6.7.1-13.4,0-26.8-.1-40.2,0-5.4,0-10.8.1-16.3Z" style="fill:url(#Gradiente_sem_nome_19-9)"/></svg>`

**Comment by ronaldorj** [comment]
css do botão add em static styles = css:

.globoPlayButton {
            background: #000;
          }

**Comment by PRProd** [comment]
Greetings @ronaldorj!

I have opened issue #273 for this request.  Thanks for all of that information!

-Doug
